package.json

```diff
@@ -50,13 +49,14 @@
     "trim": "^1.0.1"
   },
   "devDependencies": {
+    "decanter": "^6.2.17",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.2.1",
     "local-ssl-proxy": "^1.3.0",
     "netlify-cli": "^12.1.0",
```

Comment:
I've been moving `netlify-cli` to `optionalDependencies` lately as it isn't even required for development.

Reply:
Totally makes sense! Just pushed up the commit 👍🏼 😄